src-backbone/app/js/views/procedures/procedures_layout.js

```diff
@@ -78,7 +78,9 @@ module.exports = Marionette.LayoutView.extend({
         event.preventDefault();
 
         var self = this;
-        var new_procedure = new this.Procedure({}, {
+        var new_procedure = new this.Procedure({
+            author: 'Stephen' // TODO either fetch user's username or let API allow nullable authors
```

Comment:
Yea this is actually the correct way to initialize a model. I was actually surprised the previous way worked too